src/modules/plan_indicativo_auditoria/plan_indicativo_auditoria.service.ts

```
@@ -0,0 +1,643 @@
+import { Injectable, InternalServerErrorException } from '@nestjs/common';
+import { SupabaseService } from 'src/config/supabase/supabase.service';
+import * as ExcelJS from 'exceljs';
+import {
+    ListarSnapshotsDto,
```

ESLint

[error] Delete `··`

(prettier/prettier)